mypy/typeshed/stdlib/_typeshed/__init__.pyi

```diff
@@ -293,7 +293,7 @@ class structseq(Generic[_T_co]):
     # https://github.com/python/typeshed/pull/6560#discussion_r767149830
     def __new__(cls: type[Self], sequence: Iterable[_T_co], dict: dict[str, Any] = ...) -> Self: ...
 
-# Superset of typing.AnyStr that also inclues LiteralString
+# Superset of typing.AnyStr that also includes LiteralString
```

Comment:
The `mypy/typeshed` directory is vendored from https://github.com/python/typeshed, so typos should be fixed in the upstream repo rather than in `mypy/typeshed`.

Reply:
(I'm a maintainer over at typeshed, but here I'm only a triager.)

Yeah, I don't think it's worth it over at typeshed. Every so often I do a pass over the codebase with codespell, and it usually picks up a few typos in comments, but rarely finds any bugs. I wouldn't want the hassle of having to constantly update the codespell list of ignored words.

For mypy I can see a stronger rationale, though I'm still fairly neutral on the idea (and I'm not a maintainer, so it's not my decision anyway).

Reply:
Fixing typos is pretty uncontroversial though, so if you filed a PR just doing that (and didn't add the codespell config etc), I have high confidence that it would be accepted

Reply:
commits are all nicely separated, so an interested one to do the dance once in awhile could just cherry pick commits with typo fixes. The main effort indeed goes into making it typos-free going forward.